pkg/daemon/powershell/powershell.go

```diff
@@ -16,7 +21,7 @@ type commandRunner struct{}
 // Run runs the command with the PowerShell on PATH
 func (r *commandRunner) Run(cmd string) (string, error) {
 	out, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), cmdExitNoStatus) {
```

Comment:
Could you explain why this change is needed?

Reply:
Restart-computer always returns error with cmdExitNoStatus